src/injected/details-dialog-handler.ts

```diff
@@ -141,7 +141,7 @@ export class DetailsDialogHandler {
 
         const modal = shadowRoot.querySelector('.insights-dialog-main-override-shadow');
         if (modal != null) {
-            document.body.classList.add('insights-modal');
+            this.htmlElementUtils.getBody();
```

Comment:
This should still be adding to the classList, right?

Reply:
Ugh, yes! My bad! I'll fix it!